transactions.go

```diff
@@ -66,13 +66,16 @@ func (b *BuxClient) SendToRecipients(ctx context.Context, recipients []*transpor
 ) (*buxmodels.Transaction, transports.ResponseError) {
 	draft, err := b.DraftToRecipients(ctx, recipients, metadata)
 	if err != nil {
+		Log.Error().Err(err).Str("transactions", "SendToRecipients").Msg(err.Error())
 		return nil, err
 	} else if draft == nil {
+		Log.Error().Err(buxerrors.ErrDraftNotFound).Str("transactions", "SendToRecipients").Msg(buxerrors.ErrDraftNotFound.Error())
```

Comment:
You can use `log.Err(error).Msg("")` instead of `log.Error().Err(error).Msg("")`
I think using `.Msg(error.Error())` is redundant

Reply:
In documentation use like this. `log.Error().Err(err).Msg("")` https://github.com/rs/zerolog#leveled-logging. Without `Error` it will be debug level. Will remove `.Msg()`

Reply:
As the team we prefer error message in `message` field only, so please forget my above comments :D use syntax like:
`log.Error().Msg(error.Error())` ;-)